src/js/view/controls/components/volumetooltipicon.ts

```diff
@@ -81,14 +81,21 @@ export default class VolumeTooltipIcon extends TooltipIcon {
         this.addSliderHandlers(this.horizontalSlider.uiOver);
         this.addSliderHandlers(this.verticalSlider.uiOver);
 
-        this.onAudioMode(null, _model.get('audioMode'));
+        this.updateSlider(_model, _model.get('audioMode'));
 
-        this._model.on('change:audioMode', this.onAudioMode, this);
+        this._model.on('change:audioMode', this.updateSlider, this);
     }
 
-    onAudioMode(model: Model | null, val: boolean): void {
-        const tabIndex = val ? 0 : -1;
-        setAttribute(this.horizontalContainer, 'tabindex', tabIndex);
+    updateSlider(model: Model, audioMode: boolean): void {
+        const hasHorizontalSlider = model.get('horizontalVolumeSlider') || audioMode;
+
+        if (hasHorizontalSlider) {
+            this.horizontalContainer.style.display = 'flex';
+            this.tooltip.style.display = 'none';
```

Comment:
If possible, toggle classes rather than altering style properties. This will help separate responsibilities, as well as (negligibly) improve performance.

Reply:
Good idea! I had been trying to figure out an element to toggle the class on and I think I was able to find a good one.